sig-vmware/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG VMware Charter
+
+## Mission
+
+The VMware SIG maintains and evolves the ability to run Kubernetes on VMware infrastructure.
+
+## Scope
+
+SIG VMware serves to bring together members of the VMware and Kubernetes community to maintain, support and provide guidance for running Kubernetes on VMware platforms.
+
+One focus area is implementing and maintaining the Kubernetes cloud provider integration code for vSphere.
+
+This SIG will also fill a role beyond implementation of a cloud provider for vSphere. Thus it is appropriate to operate a VMware SIG, rather than expecting that all VMware infrastructure related activity be managed as a working group underneath sig-cloud-provider. 

```

Comment:
I don't think this is necessary as the following paragraph outlines a large scope well.